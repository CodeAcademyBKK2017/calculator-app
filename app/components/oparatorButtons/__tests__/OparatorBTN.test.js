import 'react-native';
import OPBTN from '../OparatorBTN.component';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <OPBTN />
  );
  expect(tree).toBeDefined();
});

it('BaseView: renders correctly', () => {
  const tree = renderer.create(<OPBTN />).toJSON();
  expect(tree).toMatchSnapshot();
});