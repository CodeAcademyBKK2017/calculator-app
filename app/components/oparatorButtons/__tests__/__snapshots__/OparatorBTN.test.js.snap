// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseView: renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  collapsable={undefined}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  nativeID={undefined}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "borderWidth": 1,
      "height": 30,
      "justifyContent": "space-around",
      "opacity": 1,
      "width": 70,
    }
  }
  testID={undefined}
  tvParallaxProperties={undefined}
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    
  </Text>
</View>
`;
